src/content/docs/mobile-monitoring/mobile-monitoring-ui/mobile-app-pages/version-trends-compare-user-adoption-metrics-performance.mdx

```diff
@@ -11,65 +11,42 @@ redirects:
-
-## View version trend data in UI [#view-version-trends]
-
-To view the **Version trends** report:
+* New Relic mobile agent for [Android agent v6.5.0](/docs/release-notes/mobile-release-notes/android-release-notes/) or higher
+* New Relic mobile agent for [iOS agent v7.0.0](/docs/release-notes/mobile-release-notes/ios-release-notes/) or higher
 
-1. Go to **[one.newrelic.com](https://one.newrelic.com) > Mobile > (select an app) > App > Version trends**.
-2. Select the **Launch** time period to compare versions by day (default), week, or month, or view all available data.
+## View version trends [#view-version-trends]
 
-   ![Version Trends iOS 2016-01-07 Launch buttons.png](./images/Version-Trends-iOS-2016-01-07-Launch-buttons_0.png "Version Trends iOS 2016-01-07 Launch buttons.png")
-3. Review the [version trend details](#details) to analyze performance, user adoption, and key technical indicators. For additional suggestions about how to interpret the data, review the [examples](#examples).
-4. Optional: To configure email settings, see [Email reports](/docs/mobile-monitoring/mobile-monitoring-ui/mobile-app-pages/mobile-monitoring-email-notifications#version-trends).
+To view the version trends report go to **[one.newrelic.com](https://one.newrelic.com) > Mobile > (select an app) > App > Version trends**.
 
-## View version trend details [#details]
+![A screenshot of the version trends UI that shows several app versions and the performance differences as percentages between them](./images/version-trends-ui.png "Mobile version trends UI")
 
-Each version includes several metrics to analyze performance, user adoption, and key technical indicators.
-
-<CollapserGroup>
-  <Collapser
-    id="performance"
-    title="Performance across releases"
-  >
-    An important feature is comparison of app performance across releases, based on data collected when each version was at its peak of popularity. You can select four different views of data for when each version was in production, including:
+Each new version of your mobile app is compared against the previous version. For a selected time period, the version trends report compares the performance increases and decreases of your app versions against the previous version.
 
-    * Launch day (default)
-    * Launch week
-    * Launch month
-    * All available data for the version
+For example, use the timepicker dropdown to select default or custom time periods to query:
 
-    This quickly helps you see whether the adoption and performance characteristics of an app's version are better or worse than others.
+![This is a screenshot of the time picker within the version trends UI. Dates selected are March 24th through March 25th.](./images/version-trends-time-picker.png "Mobile version trends time picker")
 
-    Summary mobile app information for each release includes:
+## Understanding version trend details [#details]
```

Comment:
We try to avoid "ing" verb form in headings. See style guide https://docs.newrelic.com/docs/style-guide/structure/levels-headings

Reply:
Thank you!